subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/results/DefaultOutputDirSelector.groovy

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.performance.results
+
+import com.google.common.base.Strings
+import groovy.transform.CompileStatic
+
+@CompileStatic
+class DefaultOutputDirSelector implements OutputDirSelector {
+    private static final String DEBUG_ARTIFACTS_DIRECTORY_PROPERTY_NAME = "org.gradle.performance.debugArtifactsDirectory"
```

Comment:
I know this property is from legacy code, but do you see anywhere we set it? I searched a little bit but didn't find anywhere to set it. If nobody is setting it I think we can totally remove this one.

Reply:
Yes, we still use this property, and it is important: https://github.com/gradle/gradle/blob/ed9d43311b646d05bbd65a88db1a38b8e015fa30/build-logic/performance-testing/src/main/groovy/gradlebuild/performance/tasks/PerformanceTest.groovy#L320